src/Angor/Client/Pages/Create.razor

```diff
@@ -194,6 +260,53 @@
         }
 
         await _RelayService.ConnectToRelaysAsync();
+
```

Comment:
This will happen every time we land on the create page, is that correct? or we want to connect on landing on the site? is it idempotent?

Reply:
It basically checks if there is a connection otherwise opens it.